governance/xc_admin/packages/xc_admin_frontend/package.json

```diff
@@ -9,9 +9,9 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "@coral-xyz/anchor": "^0.26.0",
+    "@coral-xyz/anchor": "^0.28.1-beta.1",
```

Comment:
this has to be the same as the version of `@coral-xyz/anchor` that `@pythnetwork/client` is using